### PR TITLE
add new blog of openfunction

### PR DESCRIPTION
**Pull request by faweizhao26**
Signed-off-by: faweizhao26 <faweizhao@kubesphere.io>

### CODE REVIEW

File: content/zh/blogs/faas-openfunction.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
+---
+title: '从 0 到 1，打造新一代开源函数计算平台'
```
Comment (yangchuansheng) [inline content/zh/blogs/faas-openfunction.md:2]: OpenFunction: 从 0 到 1，打造新一代开源函数计算平台

Comment (faweizhao26) [reply content/zh/blogs/faas-openfunction.md]: Done.

**Pull request merged**